sources/2023-09-19-initialize-reward-rate-configuration/initialize.move

```diff
@@ -0,0 +1,20 @@
+script {
+    use aptos_framework::aptos_governance;
+    use aptos_framework::staking_config;
+    use aptos_std::fixed_point64;
+
+    fun main(proposal_id: u64) {
+        let framework_signer = aptos_governance::resolve_multi_step_proposal(proposal_id, @0000000000000000000000000000000000000000000000000000000000000001, vector::empty<u8>());
+        let (curr_rewards_nominator, curr_rewards_denominator) = staking_config::get_reward_rate(&staking_config::get());
+        staking_config::initialize_rewards(
```

Comment:
`staking_config::initialize_rewards` doesn't seem to exist either.

Reply:
same as above